src/Api/Models/Response/SelectionReadOnlyResponseModel.cs

```diff
@@ -4,7 +4,7 @@ namespace Bit.Api.Models.Response;
 
 public class SelectionReadOnlyResponseModel
```

Comment:
Small question, should we consider changing this to `CollectionAccessResponseModel`? Or did you want to leave it more abstract?

Reply:
Must've missed this when I was renaming the core model, nice find! I'll update when blockers for this PR have been resolved :)